src/pint/toa.py

```diff
@@ -567,15 +569,15 @@ def _parse_TOA_line(line, fmt="Unknown"):
 
 
 def format_toa_line(
-    toatime,
-    toaerr,
-    freq,
-    obs,
-    dm=0.0 * pint.dmu,
-    name="unk",
-    flags={},
-    format="Princeton",
-    alias_translation=None,
+    toatime: time.Time,
+    toaerr: u.Quantity,
+    freq: u.Quantity,
+    obs: Observatory,
+    dm: u.Quantity = 0.0 * pint.dmu,
+    name: str = "unknown",
+    flags: dict = {},
+    format: str = "Princeton",
+    alias_translation: dict = None,
```

Comment:
I thought that if the default was `None`, it needed an `Optional` tag since `None` is not a `dict`?
(don't know if that is present elsewhere too).

Reply:
Added Optional